x-pack/metricbeat/module/mssql/db/data.go

```diff
@@ -0,0 +1,25 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package db
+
+import (
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstrstr"
+)
+
+var (
+	schema = s.Schema{
+		"database": s.Object{
+			"id": c.Str("database_id"),
+		},
+		// Returns space usage information for the transaction log.
+		"log_space_usage": s.Object{
+			"total.bytes":             c.Int("total_log_size_in_bytes"),
```

Comment:
You need to use the `s.Object` here to have in the end `total` as an object and `bytes` as the key. Same as you do on line 18.

Reply:
Also done, but I'd like some review

Reply:
LGTM